pandas/util/_decorators.py

```diff
@@ -5,6 +5,21 @@
 from textwrap import dedent, wrap
 from functools import wraps, update_wrapper, WRAPPER_ASSIGNMENTS
 
+def validate_kwarg():
+    def _validate_kwarg(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            expected_keys=['io', 'sheet_name','header', 'names', 'index_col', 'usecols', 'squeeze', 'dtype', 'engine', 
+               'converters', 'true_values',  'false_values', 'skiprows', 'nrows', 'na_values', 'verbose', 'parse_dates',
+               'date_parser', 'thousands', 'comment', 'skipfooter', 'convert_float']
+
+            if set(kwargs.keys()).difference(set(expected_keys)): 
+                raise ValueError('invalid parameter found')
```

Comment:
Would be nice to be explicit about which parameter isn't working. Also want a test